xml/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.xml

```diff
@@ -212,7 +212,7 @@
         <param name="hostBuilder">The <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> to configure.</param>
         <param name="cancellationToken">A <see cref="T:System.Threading.CancellationToken" /> that can be used to cancel the console.</param>
         <summary>Enables console support, builds and starts the host, and waits for Ctrl+C or SIGTERM to shut down.</summary>
-        <returns>A <see cref="T:System.Threading.Tasks.Task" /> that only completes when the console is cancelled or shutdown is triggered.</returns>
+        <returns>A <see cref="T:System.Threading.Tasks.Task" /> that only completes when the token is triggered or shutdown is triggered.</returns>
```

Comment:
I wonder if "triggered" is the right word for when cancellation is requested.

https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtokensource?view=netcore-3.1 uses the term "signals", so maybe "signaled"?